src/lib/flight_tasks/tasks/Offboard/FlightTaskOffboard.cpp

```diff
@@ -73,6 +68,8 @@ bool FlightTaskOffboard::update()
 	// reset setpoint for every loop
 	_resetSetpoints();
 
+	_sub_triplet_setpoint.update();
```

Comment:
There's a race condition with offboard that we keep hitting in MAVSDK testing. Pushing the setpoint triplet update as late as possible isn't really a fix, but it provides a bit more time before activation failure.